pyrokinetics/gk_code/GKOutputReaderCGYRO.py

```diff
@@ -116,6 +125,14 @@ def _init_dataset(
             xr.Dataset: Dataset with coords and attrs set, but not data_vars
         """
 
+        # Coordinate units for pyro and cgyro
+        pyro_coord_units = coord_units(local_norm)
+        cgyro_coord_units = coord_units(local_norm.cgyro)
+
+        bunit_over_b0 = (
+            (1 / cgyro_coord_units["ky"]).to(local_norm.pyrokinetics).magnitude
```

Comment:
Just to note that `.to(local_norm)` converts to the default convention, i.e. `pyrokinetics`, but I guess it's also less explicit